subsys/bluetooth/host/audio/iso.c

```diff
@@ -655,8 +650,10 @@ static int bt_iso_setup_data_path(struct bt_conn *conn)
 	int err;
 	struct bt_iso_chan *chan;
 	struct bt_iso_chan_path path = {};
-	struct bt_iso_data_path out_path = { .dir = BT_ISO_CHAN_CTRL_TO_HOST };
-	struct bt_iso_data_path in_path = { .dir = BT_ISO_CHAN_HOST_TO_CTRL };
+	struct bt_iso_data_path out_path = {
+		.dir = BT_HCI_DATAPATH_DIR_CTLR_TO_HOST };
+	struct bt_iso_data_path in_path = {
+		.dir = BT_HCI_DATAPATH_DIR_HOST_TO_CTLR };
```

Comment:
Can we use consistently direction host_to_ctlr, ctlr_to_host and drop using input/output terms so that there is no mixed usage. Or stick to input and output, which is what spec uses for direction parameter value.

Reply:
This PR does not change the terminology, so I consider this discussion outside its scope.